ports/rp2/boards/WAVESHARE_RP2040_LCD_0_96/board.json

```diff
@@ -0,0 +1,21 @@
+{
+    "deploy": [
+        "../deploy.md"
+    ],
+    "docs": "",
+    "features": [
+        "Dual-core",
+        "Battery Charging",
+        "External Flash",
+        "Display",
+        "USB-C"
+    ],
+    "images": [
+        "rp2040-lcd-0.96-1.jpg"
```

Comment:
this image is not provided in this PR?

Reply:
Board images are intended to be submitted as a separate PR on the micropython-media repo, they're not included in this main repo to keep the size down.

Reply:
image is provided in this PR(https://github.com/micropython/micropython-media/pull/62)